pkg/webhook/pod/pd_deleter.go

```diff
@@ -102,7 +87,7 @@ func (pc *PodAdmissionControl) admitDeleteNonPDMemberPod(IsDeferDeleting, isInOr
 		// it would be existed an pd-3 instance with its deferDeleting label Annotations PVC.
 		// And the pvc can be deleted during upgrading if we use create pod webhook in future.
 		if !isInOrdinal {
-			err := addDeferDeletingToPVC(v1alpha1.PDMemberType, pc, tc, ownerStatefulSet.Name, namespace, ordinal)
+			err := addDeferDeletingToPVC(v1alpha1.PDMemberType, pc, payload.tc, ownerStatefulSet.Name, namespace, ordinal)
```

Comment:
If pass `pc` as an argument, why not `pc. addDeferDeletingToPVC `?

Reply:
BTW, `v1alpha1.PDMemberType, ownerStatefulSet.Name, ordinal` are only used to construct the PVC name, is it better to construct the name and pass the name as an argument?

Reply:
updated.